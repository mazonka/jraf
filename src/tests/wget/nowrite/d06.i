command=jw data 0 /tmp/d2/file_yes 1 Kg=