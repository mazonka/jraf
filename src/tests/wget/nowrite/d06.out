JRAF_FAIL deny data